packages/core/src/codewhisperer/util/authUtil.ts

```diff
@@ -186,15 +195,19 @@ export class AuthUtil implements IAuthProvider {
     async loginIam(
         accessKey: string,
         secretKey: string,
-        sessionToken?: string
+        sessionToken?: string,
```

Comment:
As mentioned above, may be worth to refactor in to a single object as the argument